src/_gettsim/transfers/kindergeld.py

```diff
@@ -174,3 +176,39 @@ def kind_bis_10_mit_kindergeld(
     """
     out = kindergeld_anspruch and (alter <= 10)
     return out
+
+
+@policy_info(skip_vectorization=True)
+def gleiche_fg_kindergeldempfänger_kind(
```

Comment:
```suggestion
def same_fg_as_kindergeldempfänger(
```
I think this should rather be English/Denglisch? Else `gleiche_fg_wie_kindergeldempfänger`, but in any case, the name should make it obvious that we are looking from the child level rather than symmetrically.